src/java/main/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -122,8 +122,8 @@
     final Map<String, ChangeRecord> outstandingChangesForPath =
         new HashMap<String, ChangeRecord>();
 
-    protected ServerCnxnFactory serverCnxnFactory;
-    protected ServerCnxnFactory secureServerCnxnFactory;
+    ServerCnxnFactory serverCnxnFactory;
```

Comment:
`protected` is removed here. Is this intended?

Reply:
other than this, patch LGTM!

Reply:
Intellij suggested. I think because it's more restrictive.